src/internal/utils.js

```diff
@@ -146,6 +146,41 @@ function canConvertValue(value) {
   )
 }
 
+/**
+ * Splits a shadow style into its separate shadows using the comma delimiter, but creating an exception
+ * for comma separated values in parentheses often used for rgba colours.
+ * @param {String} value
+ * @returns {Array} array of all box shadow values in the string
+ */
+function splitShadow(value) {
+  const shadows = []
+  let start = 0
+  let end = 0
+  let rgba = false
+  while (end < value.length) {
+    if (!rgba && value[end] === ',') {
+      shadows.push(value.substring(start, end).trim())
+      end++
+      start = end
+    } else if (value[end] == `(`) {
```

Comment:
There are a couple lines that will probably fail eslint because I prefer triple equals.